### PR TITLE
Add gcp profile for hive long duration cases

### PR DESCRIPTION
**Pull request by jianping-shu**
https://issues.redhat.com/browse/OCPQE-12572
Add gcp profile for Hive long duration tests cases

### CODE REVIEW

**Comment by kuiwang02** [comment]
please get qe productivity team to approve it.

**Comment by jianping-shu** [comment]
@lwan-wanglin @huangmingxia @LiangquanLi930 @heliubj18 This is the PR I just sent you in the slack for review, thanks

**Comment by jianping-shu** [comment]
@lwan-wanglin @huangmingxia @LiangquanLi930 @heliubj18 This is the PR I just sent you in the slack for review, thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -1139,6 +1139,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-upi-xpn
+- as: e2e-gcp-ipi-hive-p3
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:1142]: Please sort the jobs on this line.

Comment (jianping-shu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml]: Done, thanks

**Comment by jianping-shu** [comment]
@liangxia Rename the gcp profile as suggested, PTAL, thanks

**Comment by liangxia** [comment]
@jianping-shu Would you mind to squash the commits into one commit ?

**Comment by jianping-shu** [comment]
OK

**Pull request merged**